vendor/install-extensions.js

```diff
@@ -2,8 +2,12 @@ function installExtensions() {
   try {
     window.requireNode('fs');
   } catch (e) {
-    console.warn('ember-electron is unable to require node modules, possibly because nodeIntegration is not enabled.');
-    console.warn('This prevents the installation of the Ember Inspector and Devtron extensions. Error:');
+    console.warn(
+      'ember-electron is unable to require node modules, possibly because nodeIntegration is not enabled.'
+    );
+    console.warn(
+      'This prevents the installation of the Ember Inspector and Devtron extensions. Error:'
```

Comment:
It no longer prevents the installation of Ember Inspector! 🎉

Reply:
It might still prevent it though right? I don't know what electron-devtools-installer does behind the scenes, but I wasn't sure if maybe this was still relevant?

Reply:
No, I don't think so. The only reason that's a problem for this code is that here we use the remote module from the renderer process to install it, so if node integration is disabled, the remote module won't work.

But now we're doing it from the main process, so node integration has nothing to do with it, and if something causes an error installing it, that will show up in the error log in the catch block of `installExtensions(EMBER_INSPECTOR)`.

So let's change this to just talk about Devtron, since that's all we're installing here via the remote module after this change.